src/fontra/client/core/glyph-controller.js

```diff
@@ -557,10 +557,16 @@ export class StaticGlyphController {
   }
 
   get leftMargin() {
+    if (this.bounds === undefined) {
```

Comment:
I prefer the `?` spelling here. `undefined` is fine (and perhaps indeed better than `null`).

Reply:
Changed it.